### PR TITLE
Add Use Takefocus plugin

### PR DESCRIPTION
**Pull request by ImperatorS79**
So that @Kreyren can actually use it ;-).

### CODE REVIEW

File: Engines/Wine/Plugins/UseTakeFocus/script.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "scriptName"                 : "Use Take Focus",
+    "id"                         : "engines.wine.plugins.usetakefocus",
+    "compatibleOperatingSystems" : [
+        "MACOSX"
```
Comment (plata) [inline Engines/Wine/Plugins/UseTakeFocus/script.json]: only for OSX?

Comment (Kreyren) [reply Engines/Wine/Plugins/UseTakeFocus/script.json]: ~~triggered~~

**Comment by Kreyren** [comment]
Wait what? I need take focus anywhere? :thinking:

**Comment by ImperatorS79** [comment]
@Kreyren in MTG script ?

**Comment by Kreyren** [comment]
> @Kreyren in MTG script ?

https://github.com/PhoenicisOrg/scripts/pull/939 ? :thinking:

**Comment by ImperatorS79** [comment]
That pull was a setting, for the GUI. Here is the plugin, for use inside a script, If I understood correctly how it works.

**Comment by Kreyren** [comment]
Guess i did gut then :thinking: One less problem i can't be blame about yaay

**Pull request merged**